aiosmtpd/handlers.py

```diff
@@ -13,11 +13,13 @@
 import smtplib
 
 from email import message_from_bytes, message_from_string
+from email.feedparser import NLCRE
 from public import public
 
 
+EMPTYSTRING = ''
 COMMASPACE = ', '
-NEWLINE = '\n'
+NEWLINE = '\r\n'
```

Comment:
Can we call this module global `CRNL`?  We might need a pure `\n` later and that we would call `NL` or `NEWLINE`.